src/docs/faqs/faqs.md

```diff
@@ -1,48 +1,55 @@
 ---
 layout: docs
 title: Frequently Asked Questions
-published: 2022-12-14
+published: 2023-05-11
 add toc: true 
 add support: true 
 add sidebar: _sidebars/mainSidebar.md
 article_status: published
-todo: 
-1. Search for existing Questions (CMR)
-2. Provide answers (CMR)
-3. Include links to corresponding KB article (CMR)
-4. Open FAQ-section to public (CMR)
 ---
 
 ## General
 
-**Q:**  
-
-**A:**  
+<details><summary><span style="color: #1fc2a7;font-size:1.2em">
+Which format must be used to enter a date?
```

Comment:
I'd recommend to avoid words like must, should, require, need to. Especially for the first question popping up here. 
Also, since it's under "general", it sounds like one always **must** use the iso standard, but I can very well use other formats e.g. in free-text protocols for human readability.

Reply:
I think this depends on the point of view. If we look at our guideline that everythin is possible you are correct, but ISA actually requires the ISO standard

Reply:
Right, see comment below. I'd add this to ISA-specific questions then.

Reply:
See comment below.